### DISCUSSION TITLE
Resetting --time-cond more than once in a curl --config file

### DISCUSSION THREAD

**Discussion by jidanni** [curl/curl#10698] [Q&A] [answered]
Here's my curl --config:
```
url = A
output = B
url = C
output = D
url = E
output = F
...
```
Today I want to soup it up by adding --time-cond!:
```
url = A
output = B
time-cond = B
url = C
output = D
time-cond = D
url = E
output = F
time-cond = F
...
```

But unlike --output, the man page says "if --time-cond is provided
several times, the last set value will be used."

So does that my dream of just invoking curl just once to fetch my 145 files
is over, and I must invoke it 145 times?

But wait, [maybe](https://github.com/curl/curl/discussions/10697) it doesn't read the whole config file before it starts
its fetching, and I indeed can reset time-cond for each fetch?

**Comment by bagder** [comment, marked as answer]
It means you need a put a `--next` between them so that you can use `time-cond` again, "fresh".
